src/utils/auth-methods.ts

```diff
@@ -115,5 +115,21 @@ export const PROVIDERS : Provider[] = [
       }
     ],
     page: '/verify/e-ids/belgian-eid'
+  },
+  {
+    title: 'Age verification',
+    authMethods: [
+      {
+        title: 'Age verification',
+        acrValue: 'urn:age-verification',
+        scopes: [
```

Comment:
At the moment this does not match the available scopes on master.

The current ones are: 15, 16, 18, 21 (https://github.com/criipto/extensions-service/blob/master/extensions/ageverification/index.ts#L337)